src/ethereum/base_types.py

```diff
@@ -315,7 +340,8 @@ def wrapping_add(self, right: int) -> "U256":
         if result == NotImplemented:
             return NotImplemented
 
-        result %= 2 ** 256
+        # This is a fast way of ensuring that the result is < (2 ** 256)
+        result &= self.MAX_VALUE
```

Comment:
I understand that this might not be straight forward to grasp at an initial glance. That's the reason I added the comment. It should help the readers given the comment.